packages/chart/src/FigureChartModel.ts

```diff
@@ -546,10 +616,10 @@ class FigureChartModel extends ChartModel {
    * value for x.
    * @param {dh.Series} series The series to clean the data for
    */
-  cleanSeries(series) {
+  cleanSeries(series: Series): void {
     const { name, plotStyle } = series;
     const seriesData = this.seriesDataMap.get(name);
-    if (plotStyle === dh.plot.SeriesPlotStyle.HISTOGRAM) {
+    if (seriesData != null && plotStyle === dh.plot.SeriesPlotStyle.HISTOGRAM) {
```

Comment:
Just return early if seriesData is `null`, then you don't need the assert/if null checks here

```
const seriesData = this.seriesDataMap.get(anem);
if (seriesData == null) {
  return;
}
...
```